tests/pytorch/nightly/resnet50-mp.libsonnet

```diff
@@ -80,6 +80,34 @@ local tpus = import 'templates/tpus.libsonnet';
       },
     },
   },
+  // DDP converges worse than MP.
+  local convergence_ddp = common.Convergence {
```

Comment:
Can you refactor this to inherit from `convergence`? It would look something like

```
local convergence_ddp = convergence {
  metricConfig+: {
    ...
  }
}
```